dist/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/162dfb3961.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Lists To DO</title>
+  <script defer src="main.js"></script></head>
+  <body>
+    <section class="main">
+      <div class="header">
+        <h1 class="heading">To Do Lists</h1>
+        <i class="fa-solid fa-rotate"></i>
+      </div>
+      <form action="">
+        <div class="inputcontainer">
+          <input type="text" class="input" placeholder="Add your list..." />
+        </div>
```

Comment:
[ Suggested Change ]
You did well so far on your to-do list app structure. There is one more addition I would like for you to consider. It's not that major to ask you for a required change but I'm certain you'll consider adding the enter icon to the left of the input field in order for your design to look similar to the original sample given. 

- **The original design**
![image](https://user-images.githubusercontent.com/50941074/192647404-0e0a2d66-c108-4412-bb99-93042b4b5f7f.png)

**Your Design**
![image](https://user-images.githubusercontent.com/50941074/192647500-7d780e90-cc92-4e4b-ac82-dbaac1f8bfea.png)

Reply:
I will Thank you @AmaduKamara